manifests/base/jupyter-habana-notebook-imagestream.yaml

```diff
@@ -1,9 +1,9 @@
 ---
+# This Image is Deprecated and will not be displayed on the Dashboard
+# Keeping this here for satisfying upgrade requirements
 apiVersion: image.openshift.io/v1
 kind: ImageStream
 metadata:
-  labels:
-    opendatahub.io/notebook-image: "true"
```

Comment:
```
opendatahub.io/notebook-image: "false"
```

This is a suggestion from

* https://github.com/red-hat-data-services/notebooks/pull/404#pullrequestreview-2365619187

and I think it's a good one

Reply:
Yeah, we could do it that way, and yes seems more correct. (Sorry,I missed the comment from the other PR)

Reply:
Thank you for pointing out, I changed it.